www/sass/modules/_base.scss

```diff
@@ -12,7 +12,8 @@
 $color-white: #fff; // white
 
 // status colors and text
-$state-colors: 
```

Comment:
Also if you change *.scss files, you need to regenerate and include to commit `prod/css/default.css`.

Reply:
Done with `grunt build prod`